templates/acs-deployment-master.yaml

```diff
@@ -370,11 +370,10 @@ Parameters:
       Description: The Admin password for Alfresco
       NoEcho: True
     RegistryCredentials:
-      AllowedPattern: ".+"
-      ConstraintDescription: The private registry credentials can not be empty
       Type: String
-      Description: The private registry credentials base64
+      Description: The private registry credentials base64. The default refers to a non existent private registry.
       NoEcho: True
+      Default: "ewoJImF1dGhzIjogewoJCSJodHRwczovL3ByaXZhdGVyZWdpc3RyeS5jb20iOiB7CgkJCSJhdXRoIjogImRHVnpkRUIwWlhOMExtTnZiVHB3WVhOemQyOXlaQW89IgoJCX0sCgl9LAoJIkh0dHBIZWFkZXJzIjogewoJCSJVc2VyLUFnZW50IjogIkRvY2tlci1DbGllbnQvMTguMDYuMS1jZSAoZGFyd2luKSIKCX0sCgkic3RhY2tPcmNoZXN0cmF0b3IiOiAia3ViZXJuZXRlcyIKfSUK"
```

Comment:
I see your comment that this is pointing to a fictional private registry but it "looks" like real credentials and might be confusing for someone debugging, I would suggest using an empty string or something obvious like "replace-me"!?

Reply:
Ah I see this was a suggestion from Sergiu ;-)

Reply:
Yup. Otherwise the creation of the secret would fail

Reply:
I'm not sure about that, I tried yesterday using a value of "false" and it worked for me.

My only concern about adding something that looks real is that it sets a precedent that credentials can be committed, which is a dangerous path to go down.

Ideally having a default empty string can act as a flag to the rest of the templates i.e. only include the registry credentials if the value is not empty, but I realise that could be too much re-work.

Reply:
Yes it worked for me as well with letting the registry empty. Sergiu agreed to let it empty to avoid confusing.